app/assets/javascripts/cookbookDeprecate.js

```diff
@@ -28,8 +28,4 @@ $(document).on('opened', '[data-reveal]', function () {
   }
 
   $('.cookbook-deprecate').select2(settings);
-
-  $('.cookbook-deprecate').on("select2-selecting", function(e) {
-    $('.submit-deprecation').prop('disabled', false);
-  });
```

Comment:
Remove the now-unnecessary JS to enable the "Deprecate" submit button when something is selected in the text box.